libs/movie/src/lib/movie/components/movie-analytics-chart/utils.ts

```diff
@@ -0,0 +1,71 @@
+import { Analytics } from "@blockframes/analytics/+state/analytics.firestore";
+import { MovieAnalytics, MovieEventAnalytics } from "@blockframes/movie/+state/movie.firestore";
+import { AnalyticsEvents } from "@blockframes/utils/analytics/analytics-model";
+import { addDays } from 'date-fns'
+import { toYMD } from "./movie-analytics-chart.component";
+
+export function toMovieAnalytics(events: Analytics<'title'>[]): MovieAnalytics[] {
+  const today = new Date();
+  const current = addDays(today, -28);
+  const past = addDays(today, -56);
```

Comment:
Prefere `subDays` instead of `addDays`, it's more readable: 
```suggestion
  const current = subDays(today, 28);
  const past = subDays(today, 56);
```